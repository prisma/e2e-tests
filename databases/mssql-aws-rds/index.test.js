const { PrismaClient } = require('@prisma/client')

const prisma = new PrismaClient()

describe('tests for mssql database', () => {
  afterAll(() => {
    prisma.$disconnect()
  })

  it('should query the database', async () => {
    const data = await prisma.user.findOne({
      where: { email: 'alice@prisma.io' },
      select: { email: true, name: true },
    })
    expect(data).toMatchSnapshot()
  })
})
