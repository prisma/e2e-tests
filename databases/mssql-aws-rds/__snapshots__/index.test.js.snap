// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for mssql database should query the database 1`] = `
Object {
  "email": "alice@prisma.io",
  "name": "Alice",
}
`;
